myia/operations/ops_dunder.py

```diff
@@ -164,6 +164,7 @@ def protocol(x, y):
     name='or', lattr='__or__', rattr='__ror__',
     pyop=operator.or_
 )
+
```

Comment:
Why are you separating xor from its friends? :(
```suggestion
```

Reply:
Useless line removed!